sdk/batch/Azure.ResourceManager.Batch/src/autorest.md

```diff
@@ -44,13 +45,189 @@ rename-rules:
   SSO: Sso
   URI: Uri
   Etag: ETag|etag
+  AAD: Aad
+  LRS: Lrs
+  TCP: Tcp
+  UDP: Udp
+  NFS: Nfs
 
 prepend-rp-prefix:
 - StorageAccountType
-- ImageReference
+- ProvisioningState
+
+override-operation-name:
+  Location_CheckNameAvailability: CheckBatchNameAvailability
+  Location_GetQuotas: GetBatchQuotas
+  Location_ListSupportedCloudServiceSkus: GetBatchSupportedCloudServiceSkus
+  Location_ListSupportedVirtualMachineSkus: GetBatchSupportedVirtualMachineSkus
 
 rename-mapping:
-  DataDisk: BatchVirtualMachineDataDisk
-  NetworkProfile: BatchVirtualMachineNetworkProfile
   CachingType: BatchDiskCachingType
+  Application: BatchApplication
+  ApplicationPackage: BatchApplicationPackage
+  ApplicationPackage.properties.lastActivationTime: LastActivatedOn
+  PublicNetworkAccessType: BatchPublicNetworkAccess
+  BatchAccount.properties.dedicatedCoreQuotaPerVMFamilyEnforced: IsDedicatedCoreQuotaPerVmFamilyEnforced
+  DetectorResponse: BatchAccountDetector
+  OutboundEnvironmentEndpoint: BatchAccountOutboundEnvironmentEndpoint
+  EndpointDependency: BatchAccountEndpointDependency
+  EndpointDetail: BatchEndpointDetail
+  Pool: BatchAccountPool
+  Pool.properties.lastModified: LastModifiedOn
+  Pool.properties.provisioningStateTransitionTime: provisioningStateTransitOn
+  PoolAllocationMode: BatchAccountPoolAllocationMode
+  AutoScaleRun: BatchAccountPoolAutoScaleRun
+  ResizeOperationStatus: BatchResizeOperationStatus
+  ComputeNodeDeallocationOption: BatchNodeDeallocationOption
```

Comment:
Nit: can we avoid using `Option` here?

Reply:
can be `Setting`

Reply:
it is an enumeration. I believe the word we need to avoid is `Options` which might collide with the option bag. But option I believe it is OK